.eslintrc.js

```diff
@@ -77,5 +76,8 @@ module.exports = {
     '@typescript-eslint/no-non-null-assertion': 'error',
     'jest/no-disabled-tests': 'error',
     'jest/no-commented-out-tests': 'error',
+    'new-cap': 'off',
+    'no-self-assign': 'off',
+    'react/jsx-no-undef': 'off',
```

Comment:
please revert.
If you want, you can always disable using a comment like:

// eslint-disable-next-line @typescript-eslint/no-explicit-any